src/material/button/button.spec.ts

```diff
@@ -315,6 +312,35 @@ describe('MDC-based MatButton', () => {
         'Expected a disabled a[mat-button] not to have an enabled ripple',
       );
     });
+
+    it('should render the ripple once it is referenced', () => {
+      const fab = fixture.debugElement.query(By.css('button[mat-fab]'))!;
+      let ripple = fab.nativeElement.querySelector('.mat-mdc-button-ripple');
+      expect(ripple).toBeNull();
```

Comment:
```suggestion
      expect(ripple)
          .withContext('Expect ripple to be absent before user interaction')
          .toBeNull();
```
optional: I like adding context to assertions since it makes failures easier to read in the results
(similar for other asserts)